modules/nf-core/sylph/profile/main.nf

```diff
@@ -21,12 +21,13 @@ process SYLPH_PROFILE {
     script:
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
+    def input = meta.single_end ? "${reads}" : "-1 ${reads[0]} -2 ${reads[1]}"
```

Comment:
Does single reads not need an explicit flag?

Reply:
No, according to the documentation. I updated the PR.